### PR TITLE
Change color and position of the Outstanding balance

### PR DESCRIPTION
**Pull request by SimonaGerbocova**
Increase visibility of the Outstanding balance - change its position and color by value
Reduce width of vacuumpayroll logo and font size of links in navigation bar

### CODE REVIEW

File: client/src/pages/PayrollPage.js [hunk 1]
```diff
@@ -34,34 +34,60 @@ class PayrollPage extends Component {
...
+    const {data} = this.props
+
+    const filteredData = this.filterData(data)
+
     const oldestDate = minDate(data.map((item) => item.date))
 
     return (
       <Layout header={(
         <Fragment>
+          {this.renderBalance()}
```
Comment (paldys) [inline client/src/pages/PayrollPage.js]: You can pass `filteredData` as an argument to `renderBalance`.

File: client/src/components/Navigation.js [hunk 2]
```diff
@@ -48,11 +48,11 @@ class Navigation extends Component {
     return (
       <AppBar position="static" color="default">
         <Toolbar>
-          <div className={classes.logoContainer}>
-            <img src={Logo} alt="Logo" width="250" />
+          <div className={classes.logoContainer} style={{display: 'flex', paddingRight: '30px', borderRight: '1px solid #ccc'}}>
```
Comment (paldys) [inline client/src/components/Navigation.js]: We already use css classes here. Please, can you add these properties there (`styles` object on line 16)?

File: client/src/components/Navigation.js [hunk 3]
```diff
@@ -48,11 +48,11 @@ class Navigation extends Component {
...
+            <img src={Logo} alt="Logo" width="170" />
           </div>
 
-          <div className={classes.navigation}>
+          <div className={classes.navigation} style={{marginTop: '3px', marginLeft: '13px'}}>
```
Comment (paldys) [inline client/src/components/Navigation.js]: Same here.

File: client/src/pages/SharesPage.js [hunk 4]
```diff
@@ -60,8 +60,11 @@ class SharesPage extends Component {
     return (
       <Layout header={(
         <Fragment>
+          <div style={{marginLeft: '16px', borderBottom: '1px solid #ccc'}}>
+            <span style={{marginRight: '8px'}}>Balance:</span>
+            <span style={{fontSize: '18px', fontWeight: 'bold'}}>{balance}</span> shares
+          </div>
```
Comment (paldys) [inline client/src/pages/SharesPage.js]: Shares and Payroll pages have similar styling. Wouldn't it be better to introduce css classes for them so they can share the styling? Or are they defined separately on purpose?

File: client/src/pages/PayrollPage.js (hunk 1 above)
Comment (SimonaGerbocova) [reply client/src/pages/PayrollPage.js]: done :heavy_check_mark:

File: client/src/components/Navigation.js (hunk 2 above)
Comment (SimonaGerbocova) [reply client/src/components/Navigation.js]: done :heavy_check_mark:

File: client/src/components/Navigation.js (hunk 3 above)
Comment (SimonaGerbocova) [reply client/src/components/Navigation.js]: done :heavy_check_mark:

File: client/src/pages/SharesPage.js (hunk 4 above)
Comment (SimonaGerbocova) [reply client/src/pages/SharesPage.js]: Yes, they are similar styling but not the same for all of them. I created styles objects for them - I think this is a little better solution :slightly_smiling_face:

**Pull request merged**